### PR TITLE
MHR Transfer Framework+

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#13607

*Notes:* - This includes some work that was captured in ticket:  /bcgov/entity#13609
- Imports HomeOwner components, minor updates to integrate 
- Still requires adaptation to Transfer State and TransferValidation State
- State also might not adhere to API Schema - That also needs to be updated in above ticket.

*Description of changes:*
* Implements Transfer Route and View
* Updated Breadcrumbs
* refactored StackButtons to allow reactivity
* Updated Registrations Table to include and handle OPENING an MHR actions
* Minor styling refactors
* Updates for Fee Summary to include Transfer of Ownership
* Started Basic composable to MHR Information changes (starting with transfers) - Home for Transfer State Management
* Implements basic STATE for Transfer filing and basic Transfer validation state
* Minor store refactors - rearrangements and implements actions/mutations for base Mhr Information

* Implements HomeOwners and Review table in MHR Transfer Flow
* Minor updates to HomeOwners to integrate into Transfer flow

* Unit Test Updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  //   'getMhrRegistrationHomeDescription',
+  //   'getMhrRegistrationSubmittingParty',
+  //   'getMhrRegistrationDocumentId',
+  //   'getMhrAttentionReferenceNum',
+  //   'getMhrRegistrationLocation',
+  //   'getMhrRegistrationHomeOwnerGroups',
+  //   'getCertifyInformation',
+  //   'getStaffPayment'
+  // ])
+
```
Comment (ChasBoucher) [inline ppr-ui/src/composables/mhrInformation/useMhrInformation.ts]: I am assuming this code will be enabled as more features are added?

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  //     description: parseDescription()
+  //   }
+  //
+  //   if (getMhrAttentionReferenceNum.value) {
+  //     data.attentionReferenceNum = getMhrAttentionReferenceNum.value
+  //   }
+  //
+  //   return data
+  // }
+
```
Comment (ChasBoucher) [inline ppr-ui/src/composables/mhrInformation/useMhrInformation.ts]: Same coment as above.

**Review by ChasBoucher — APPROVED** [review]
Looks Good..

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts]: You bet, thanks! Forgot to push my clean up commit 👍 
Pushed now!

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts]: You bet, thanks! Same as above. 
Pushed my commit to clean up the comments

**Review by doug-lovett — APPROVED** [review]
Looks great, quick work.

File: ppr-ui/src/store/state/state-model.ts [hunk 3]
```diff
@@ -250,5 +262,19 @@ export const stateModel: StateModelIF = {
...
+      ownersValid: false
+    },
+    reviewConfirmValid: {
+      validateApp: false
+    }
+  },
+  mhrTransfer: {
+    mhrNumber: '',
+    ownerGroups: [],
+    owners: [],
```
Comment (dimak1) [inline ppr-ui/src/store/state/state-model.ts:277]: Do we need `owners` here?

**Review by dimak1 — APPROVED** [review]
Looks great! Just one comment about the state-model.

File: ppr-ui/src/store/state/state-model.ts (hunk 3 above)
Comment (cameron-eyds) [reply ppr-ui/src/store/state/state-model.ts:277]: Good question, you would know best. 
I saw `owners` in the state, is it required in order to store data from the HomeOwners component?

Comment (dimak1) [reply ppr-ui/src/store/state/state-model.ts:277]: Yea, I thought it was ported over by default, but after cleanup, we don't need the owners (no use). So we can remove it from here as well. Thanks!

Comment (cameron-eyds) [reply ppr-ui/src/store/state/state-model.ts:277]: Sounds good, thanks! Although i will remove in the next ticket. There is several State updates to happen with respect to the Transfer Schema, so it makes sense 👍

Comment (dimak1) [reply ppr-ui/src/store/state/state-model.ts:277]: Sounds good, resolved!

**Review by ChasBoucher — APPROVED** [review]
Looks Good.

**Pull request merged**